hack/release/draft-release.sh

```diff
@@ -84,7 +84,7 @@ release="${release#v}"
 major_minor_number="${release%.*}"
 branch="release-${major_minor_number}"
 tag="v${release}"
-title="Release $tag"
+title="Antrea $tag"
```

Comment:
Can we get the project name from `$repo` so it can be easily reused by other projects?
```
project=${repo##*/}
title="${project^} $tag"
```

Reply:
Done